report.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this should probably not be committed, should it?

Reply:
right, thanks for catching it